drivers/ieee802154/Kconfig.cc13xx_cc26xx

```diff
@@ -60,16 +56,12 @@ endif # IEEE802154_CC13XX_CC26XX
 
 menuconfig IEEE802154_CC13XX_CC26XX_SUB_GHZ
```

Comment:
Do we know what's going on here between SUB_GHZ and non-SUB_GHZ?  Are these really 2 different devices?

Reply:
apparently they are 2 distinct drivers